app/activeplan/homepage/contactArrangements/get.controller.js

```diff
@@ -0,0 +1,25 @@
+const { logger } = require('../../../../common/logging/logger')
+const { getSentencePlanComments } = require('../../../../common/data/sentencePlanningApi')
+const { getCommentText } = require('../../../../common/utils/getCommentText')
+
+const getContactArrangements = async (
+  { path, errors, errorSummary, body, renderInfo, params: { planId }, headers: { 'x-auth-token': token } },
+  res
+) => {
+  const renderDetails = renderInfo || {}
+  renderDetails.backurl = `${path.substring(0, path.lastIndexOf('/'))}#contact`
+  try {
+    const comments = await getSentencePlanComments(planId, token)
```

Comment:
this is only required for the else condition, so only retrieve it then. You can get contextual errors from that call too, by sticking a catch on the await before you re-throw for the outer catch.